src/utilcode/ccomprc.cpp

```diff
@@ -10,7 +10,10 @@
 #include "ndpversion.h"
 
 #include "../dlls/mscorrc/resource.h"
-#include "../dlls/mscorrc/resourcestring.h"
+#ifdef PLATFORM_UNIX
+#include "resourcestring.h"
+DECLARE_STRING_RESOURCE_TABLE(STRING_RESOURCE_NAME);
```

Comment:
A particular file is always built with the same "STRING_RESOURCE_NAME" -- ildasm will use different one.
This is just small convenience. We use this value to declare a string table here and also use it to create the first argument of "LoadResourceString" to point such string table.